app/src/main/java/com/google/samples/apps/sunflower/GardenActivity.kt

```diff
@@ -62,20 +35,7 @@ class GardenActivity : AppCompatActivity() {
 
         setContent {
             MdcTheme {
-                SunflowerApp(
```

Comment:
This implementation should not have been changed—the reason interop is used in this manner is so that the toolbar can collapse when you scroll in the plant list screen. With this change, the toolbar no longer collapses. I describe in this blog post the issue: https://medium.com/androiddevelopers/migrating-sunflower-to-jetpack-compose-f840fa3b9985

Reply:
Yes then please check #843번 PR #842 We will close the PR.